client/src/sections/resources/LuckyAdventures.tsx

```diff
@@ -0,0 +1,43 @@
+import { Text } from "@chakra-ui/react";
+import { availableAmount } from "kolmafia";
+import { $item, get } from "libram";
+import React, { Fragment } from "react";
+
+import Line from "../../components/Line";
+import Tile from "../../components/Tile";
+import { pluralJustDescItem } from "../../util/text";
+
+const luckyAdventureSources: [string, () => React.ReactNode][] = [
+  [
+    "Hermit",
+    () => {
+      const cloversAvailableToday = 3 - Number(get("_cloversPurchased", false));
```

Comment:
why convert this to number?

Reply:
VSCode was complaining that get returns something else, I think.